share/spack/docker/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3'
+version: '3.1'
```

Comment:
Any particular reason for the version bump?

Reply:
Because  I added the SL docker file I thought I should bump the version. I see now that this is the version of docker compose.